### DISCUSSION TITLE
Seeking help to fix rendering and stylesheet import errors in Hugo project

### DISCUSSION THREAD

**Discussion by SamGreenwood1** [CloudWithChris/hugo-creator#124] [Q&A] [answered]
$hugo-filecache = /tmp/hugo_cache/modules/filecache/modules/pkg/mod/github.com/cloudwithchris
I'm encountering two errors in my Hugo project and could use some guidance in resolving them. Here's a summary of the situation:

Error during rendering: When attempting to render a page in my project, an error occurs at line 55 of the template _default/search.html. The specific error is related to the execution of $.Scratch.Add, resulting in a "reflect.Value.Type on zero Value" error.
Stylesheet import error: While building the site, there is an issue with transforming the SCSS file style.main.scss. The error points to line 4 of main.scss and states that the import file bootstrap/functions cannot be found or read.

Actual error messages:
```
ERROR 2023/05/11 18:41:45 failed to render pages: render of "page" failed: 
"$hugo-filecache/hugo-creator@v0.0.27-0.20220911012120-e40988e0ccc9/layouts/_default/search.html:55:8": 
execute of template failed: template: _default/search.html:55:8: 
executing "main" at <$.Scratch.Add>: error calling Add: reflect: call of reflect.Value.Type on zero Value
```
```
Error: Error building site: TOCSS: failed to transform "style.main.scss" (text/x-scss): 
"$hugo-filecache/hugo-creator@v0.0.27-0.20220911012120-e40988e0ccc9/assets/sass/main.scss:4:1": 
File to import not found or unreadable: bootstrap/functions
```

If you have any experience with these errors or expertise in Hugo and SCSS, I would greatly appreciate your insights and any suggestions for resolving these issues.

**Comment by chrisreddington** [comment, marked as answer]
Closing as it's now discussed in #125
